tests/test_shadow_fork.py

```
@@ -76,25 +76,35 @@ def test_create_fails_if_base_branch_missing(self, tmp_path):
-            with pytest.raises(RuntimeError, match="does not exist"):
+            with pytest.raises(RuntimeError, match="Failed to create shadow branch"):
                 fork.create(base_branch="nonexistent")
 
     def test_create_warns_on_uncommitted_changes(self, tmp_path, caplog):
         """Test that create warns about uncommitted changes."""
         fork = ShadowFork(repo_root=tmp_path)
 
-        with patch.object(fork, "_run_git") as mock_git:
+        with patch.object(
+            fork, "_resolve_base_ref", return_value="main"
+        ) as mock_resolve, patch.object(fork, "_run_git") as mock_git:
```

Ruff (0.15.0)

[error] Local variable `mock_resolve` is assigned to but never used

Remove assignment to unused variable `mock_resolve`

(F841)